components/scream/scripts/test_all_scream.py

```diff
@@ -475,7 +476,7 @@ def generate_baselines(self, test):
             return False
         else:
             # Clean up the directory, by removing everything but the 'data' subfolder
-            run_cmd_no_fail("find -maxdepth 1 -not -name data ! -path . -exec rm -rf {} \;",
+            run_cmd_no_fail(r"find -maxdepth 1 -not -name data ! -path . -exec rm -rf {} \;",
```

Comment:
There's a stranded 'r' right before the string.

Reply:
> Python strings containing '' must be declared as raw or else this character will be interpreted as an escape character.